tldr/cross_file_calls/parsers/base.py

```
@@ -0,0 +1,17 @@
+"""
+Base parser interface for cross-file call analysis.
+"""
+
+from typing import Dict, List, Optional
```

Ruff (0.14.11)

`typing.Dict` is deprecated, use `dict` instead

(UP035)